pages/k8s/release-notes.md

```diff
@@ -21,6 +21,22 @@ toc: False
 
 ## What's new
 
+- Tigera EE support
+
+**CDK** extends its support for CNI solutions by adding the option of using
+[**Tigera EE**][tigera-home], the enterprise-ready alternative to Calico. Users are now able
+to deploy **CDK** with **Tigera EE** installed and subsequently configure additional
+features such as ElasticSearch and the CNX secure connectivity manager. For further
+details, please see the [**CDK** CNI documentation][tigera-docs]
```

Comment:
All of Tigera's documentation that I've seen refers to it as Tigera Secure EE, not Tigera EE. We should probably do the same.

This is fantastically written, nice work :+1:

Reply:
yup, good catch. thanks!